assets/stylesheets/main.css

```diff
@@ -1386,7 +1381,9 @@ div#portfolio_thumbs ul li a:hover{
 	padding: 0 40px;
 	border-radius:100%;
```

Comment:
It makes more sense to set border-radius : 50%, except there is any other purpose?

Reply:
I updated this to be 50%.